src/gotcha.c

```diff
@@ -161,7 +178,9 @@ GOTCHA_EXPORT enum gotcha_error_t gotcha_wrap(struct gotcha_binding_t* user_bind
 
   for (lib_iter = _r_debug.r_map; lib_iter != 0; lib_iter = lib_iter->l_next) {
     debug_printf(2, "Looking for wrapped callsites in %s\n", LIB_NAME(lib_iter));
-    FOR_EACH_PLTREL(lib_iter, gotcha_wrap_impl, lib_iter, bindings, num_actions);
+    if(libraryFilterFunc(lib_iter)){
```

Comment:
Oooof, I was not thinking straight when I wrote this code, you're right on that. Got callsite/callee mixed up again. Thanks for the catch, though I'm thrashing enough on this topic that I'll look a bit before acting on the feedback